### PR TITLE
request to add gcp-implementation-demos to main branch

### PR DESCRIPTION
**Pull request by tgaillard1**
holt skinner asked if I could add my repo to the main branch of document-ai-samples.

### CODE REVIEW

**Comment by holtskinner** [comment]
Ok, the linter checks are failing. Looks like most are format-related. 

You can use the contributing guide for information on structure and linters used 
https://github.com/GoogleCloudPlatform/document-ai-samples/blob/main/.github/CONTRIBUTING.md#code-quality-checks

**Comment by tgaillard1** [comment]
updated with linter to resolve formatting errors.

File: gcp-implementation-demos/docai-contract-demo/.env.local [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export PROJECT_ID=shared-vpc-host-254614
+export BUCKET_LOCATION=us-central1
+export CLOUD_FUNCTION_LOCATION=us-central1
+export INGRESS_SETTINGS=internal-and-gclb
+export CLOUD_FUNCTION_SERVICE_ACCOUNT=tg-docai
+export BQ_DATASET_NAME=acme_contract_parser_results
+export BQ_TABLE_NAME=doc_ai_extracted_entities
+export COMPANY_NAME=acme
```
Comment (holtskinner) [inline gcp-implementation-demos/docai-contract-demo/.env.local:8]: Don't commit project ids or other global identifiers into repositories, replace with a placeholder like `YOUR_PROJECT_ID`

File: gcp-implementation-demos/docai-contract-demo/cloud-functions/main.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        source_format=bigquery.SourceFormat.NEWLINE_DELIMITED_JSON,
+        ignore_unknown_values=True,
+        schema=[
+            bigquery.SchemaField("document_name", "STRING"),
+            bigquery.SchemaField("agreement_date", "DATE"),
+            bigquery.SchemaField("arbitration_venue", "STRING"),
+            bigquery.SchemaField("confidentiality_clause", "STRING"),
+            bigquery.SchemaField("effective_date", "DATE"),
+            bigquery.SchemaField("expiration_date", "DATE"),
+            bigquery.SchemaField("governing_law", "STRING"),
```
Comment (holtskinner) [inline gcp-implementation-demos/docai-contract-demo/cloud-functions/main.py:66]: Is there any way to change this to use a schema.json file or something? Seems like this is less reusable

**Review by holtskinner — COMMENTED** [review]
The main question I have is why these two separate applications are needed when most of the codebase seems to be the same?

Also follows a very similar structure to these existing projects

- https://github.com/GoogleCloudPlatform/document-ai-samples/tree/main/community/expense-parser-python
- https://github.com/GoogleCloudPlatform/document-ai-samples/tree/main/fraud-detection-python

I think it would make more sense to create a more general system that could use a schema file or create new fields based on the entities that are extracted.

Then this could be simplified into a single codebase with other applications using the examples.

**Comment by tgaillard1** [comment]
just verifying if anything is needed from my end to commit this PR

**Comment by holtskinner** [comment]
Sorry, I didn't notice that you made an additional comment. I'd recommend combining most of these two demos into a single demo because most of the code base is exactly the same.

**Pull request closed without merging**